itou/archive/management/commands/remove_unknown_emails_from_brevo.py

```diff
@@ -0,0 +1,104 @@
+from datetime import datetime, timedelta
+
+from django.db.models.functions import Lower
+from django.utils import timezone
+from sentry_sdk.crons import monitor
+
+from itou.archive.tasks import async_delete_contact
+from itou.users.models import User
+from itou.utils.brevo import BrevoClient, BrevoListID
+from itou.utils.command import BaseCommand, dry_runnable
+
+
+def modified_before(contact, cutoff):
+    try:
+        return datetime.fromisoformat(contact["modifiedAt"]) < cutoff
+    except (KeyError, TypeError, ValueError):
+        return False
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+        parser.add_argument(
+            "--wet-run",
+            action="store_true",
+            help="Perform the anonymization of cancelled approvals",
+        )
+        parser.add_argument(
+            "--offset",
+            type=int,
+            default=0,
+            help="Restart a failing run at the specified offset",
+        )
+
+    @monitor(
+        monitor_slug="remove_unknown_emails_from_brevo",
+        monitor_config={
+            "schedule": {"type": "crontab", "value": "0 22 1 * *"},
+            "checkin_margin": 5,
+            "max_runtime": 10,
+            "failure_issue_threshold": 2,
+            "recovery_threshold": 1,
+            "timezone": "UTC",
+        },
+    )
+    @dry_runnable
+    def handle(self, *args, wet_run, offset=0, **options):
+        self.wet_run = wet_run
+        limit = 1000
+        emails_to_delete_count = 0
+        two_years_ago = timezone.now() - timedelta(days=2 * 365)
+
+        while True:
+            with BrevoClient() as brevo_client:
```

Comment:
```suggestion
        with BrevoClient() as brevo_client:
            while True:
```
tu peux inverser les deux pour réutiliser le client ?